mne/io/ctf/ctf.py

```diff
@@ -129,6 +129,7 @@ def __init__(self, directory, system_clock='truncate', preload=False,
             if sample_info['n_samp'] == 0:
                 break
             if len(fnames) == 0:
+                # XXX: Buffer size different to c version
                 info['buffer_size_sec'] = \
                     sample_info['block_size'] / info['sfreq']
```

Comment:
Okay, seems that the bug is maybe different to what I thought at first. With ``system_clock='ignore'`` the data seems to load alright, but the stim channel is out of sync. I'll take a closer look tomorrow.